delivery.yaml

```diff
@@ -0,0 +1,28 @@
+build_steps:
+    - desc: 'Install required build software'
+      cmd: |
+        apt-get update
+        apt-get install -y make git apt-transport-https ca-certificates \
+        curl software-properties-common python-software-properties
+    - desc: 'Install go'
+      cmd: |
+        add-apt-repository -y ppa:longsleep/golang-backports
+        apt-get update
+        apt-get install -y golang-go
+    - desc: 'Install Docker'
+      cmd: |
+        curl -sSL https://get.docker.com/ | sh
+    - desc: 'Symlink sources into the GOPATH'
+      cmd: |
+        export GOPATH=$HOME/go
+        export OPERATOR_TOP_DIR=$GOPATH/src/github.com/zalando-incubator
+        mkdir -p $OPERATOR_TOP_DIR
+        ln -s $(pwd) $OPERATOR_TOP_DIR/kube-ingress-aws-controller
```

Comment:
nit: maybe also change the `OPERATOR_TOP_DIR` to another name, it's maybe a bit confusing why it's called this way.